packages/components/Link/src/Link.tsx

```diff
@@ -39,8 +39,9 @@ export const Link = compose<LinkType>({
     return (final: LinkProps, ...children: React.ReactNode[]) => {
       // the event fires twice due to native's implementation of inline link
       const { inline, ...mergedProps } = mergeProps(link.props, final);
+      const supportsInlineLink = Platform.OS !== 'android';
 
```

Comment:
Android supports inline links, this name is misleading in that case

Reply:
Updated